### PR TITLE
[Chore 163555583] Test for getting a single party with a given ID

### PR DESCRIPTION
**Pull request by nedssoft**
#### What does this PR do?
Test for fetching a single party record 

#### Description of Task to be completed?
- Write a test to check for when the ID is invalid
- Write a test for when the ID is valid
#### How should this be manually tested?
Run *npm run test*
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/163555583

### CODE REVIEW

File: src/tests/PartyTest.js [hunk 1]
```diff
@@ -79,4 +79,23 @@ describe('Test Party Endpoints', () => {
...
+        expect(res.body.error).to.eql('Not Found');
+        done();
+      });
+  });
+  it('Should return status 200 if the record exists', (done) => {
+    const id = 1;
+    chai.request(app)
+      .get(`/api/v1/parties/${id}`)
+      .end((err, res) => {
+        expect(res).to.have.status(200);
```
Comment (nouwatinjacob) [inline src/tests/PartyTest.js:97]: Nice job. However, you can ensure to assert for the response in the payload.

File: .eslintrc.json [hunk 2]
```diff
@@ -1,3 +1,19 @@
...
+      "mocha": true
+  },
+  "rules": {
+      "class-methods-use-this": 0,
+      "object-curly-newline" : 0,
+      "array-callback-return": 0,
+      "consistent-return" : 0,
+      "prefer-destructuring": 0
+  }
 }
```
Comment (nouwatinjacob) [inline .eslintrc.json:19]: Add newline to the end of the code

**Review by nouwatinjacob — APPROVED** [review]

File: .eslintrc.json (hunk 2 above)
Comment (nedssoft) [reply .eslintrc.json:19]: Ok. Thank you

File: src/tests/PartyTest.js (hunk 1 above)
Comment (nedssoft) [reply src/tests/PartyTest.js:97]: Okay. Thank you

**Pull request merged**